src/node-red/hardware/opto-22-groove-rio-7-mm2001-10.md

```diff
@@ -0,0 +1,57 @@
+---
+eleventyNavigation:
+  key: GRV-R7-MM2001-10
+  parent: Hardware
+meta:
+  title: Setting up Node-RED on Opto-22 Groov Rio R7
+  description: Learn how to install and configure Node-RED on the Opto-22 Groov Rio R7, a rugged edge I/O module for industrial applications.
+  keywords: node-red, flowfuse, Opto-22 Groov Rio, groov RIO, industrial IoT, edge computing
+image: "/node-red/hardware/images/GRV-R7-MM1001-10.jpeg"
+specifications:
+  Model: GRV-R7-MM2001-10
+  RAM: 1 GB DDR3
```

Comment:
I understood from our last 1:1 this was going to be normalized and you'd create a banner for all devices with less than 512MB to warn users?